components/EditSidebar/BaseCommit.js

```diff
@@ -36,6 +41,16 @@ class BaseCommit extends Component {
   render () {
     const { data: {loading, error, repo}, t } = this.props
     const commit = repo && repo.commit
+    let commitsBehind = null
+    if (repo) {
+      let commits = [...repo.commits]
```

Comment:
use const or skip variable, you never reassign it anyhow and only use it in one place